docs/generated/packages/react-native.json

```diff
@@ -492,8 +492,8 @@
         "properties": {
           "name": {
             "type": "string",
-            "description": "Project name.",
-            "$default": { "$source": "argv", "index": 0 }
+            "description": "Application project name to upgrade native files.",
+            "$default": { "$source": "projectName" }
```

Comment:
We can keep using `argv` and add `x-dropdown` attribute for the project name.

```

      "$default": {
        "$source": "argv",
        "index": 0
      },
      "x-dropdown": "project",
```

Reply:
done. changed.